assignment-client/src/scripts/EntityScriptServer.cpp

```diff
@@ -62,29 +62,38 @@ EntityScriptServer::EntityScriptServer(ReceivedMessage& message) : ThreadedAssig
 static const QString ENTITY_SCRIPT_SERVER_LOGGING_NAME = "entity-script-server";
 
 void EntityScriptServer::handleReloadEntityServerScriptPacket(QSharedPointer<ReceivedMessage> message, SharedNodePointer senderNode) {
-    auto entityID = QUuid::fromRfc4122(message->read(UUID_LENGTH_BYTES));
+    if (senderNode->getCanRez() || senderNode->getCanRezTmp()) {
```

Comment:
There is a bug where if you remove someone's permissions they will still be connected. I can add a note.